app/static/src/app/serialise.ts

```diff
@@ -30,20 +30,29 @@ function serialiseModel(model: ModelState) : SerialisedModelState {
     };
 }
 
-function serialiseSolutionResult(result: OdinRunResult | OdinFitResult | null): SerialisedSolutionResult | null {
+function serialiseSolutionResult(result: OdinRunResultOde | OdinFitResult | null): SerialisedRunResult | null {
     return result ? {
         inputs: result.inputs,
         hasResult: !!result.solution,
         error: result.error
     } : null;
 }
 
+function serialiseDiscreteResult(result: OdinRunResultDiscrete | null): SerialisedRunResult | null {
+    return result ? {
+        inputs: result.inputs,
+        hasResult: !!result.seriesSet,
+        error: result.error
+    } : null;
+}
+
 function serialiseRun(run: RunState): SerialisedRunState {
     return {
         runRequired: run.runRequired,
         parameterValues: run.parameterValues,
         endTime: run.endTime,
-        result: serialiseSolutionResult(run.result)
+        resultOde: serialiseSolutionResult(run.resultOde),
```

Comment:
once the dust settles we might look at how much we can smooth over here with the names and types; doing that might require thinking about if the type will ever change during the lifetime of an app (e.g., a version of the basic app that could work with either stochastic or ode depending on what the user writes)

Reply:
Yeah, sounds like the discrete result may grow to look more like the ode result anyway..